java/plugins/html-elements-plugin/src/test/java/ru/sbtqa/tag/pagefactory/pages/htmlelements/IndexPages/IndexPage.java

```diff
@@ -1,5 +1,6 @@
 package ru.sbtqa.tag.pagefactory.pages.htmlelements.IndexPages;
 
+import org.openqa.selenium.WebDriver;
 import org.openqa.selenium.support.FindBy;
 import ru.sbtqa.tag.pagefactory.HTMLPage;
 import ru.sbtqa.tag.pagefactory.PageFactory;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'ru.sbtqa.tag.pagefactory.PageFactory'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)